Fixed $tabularData on overview.php

DRSDavidSoft opened the pull request:
Apparently, In order for the latest versions of Kint to show 2-d arrays in tabular-view, no columns should have more than 50% empty cells .

DRSDavidSoft committed:
Fixed $tabularData on overview.php

Apparently, In order for the latest versions of Kint to show 2-d arrays in tabular-view, no columns should have more than 50% empty cells .

DRSDavidSoft commented:
Fixes #219.

DRSDavidSoft commented:
Is this PR still relevant for Kint 2.0 examples? Or the new plugin replaces this?

jnvsor commented:
The new plugin replaces it. It's also more strict than the old behaviour so it'll likely make your issue in #219 worse, though you can mitigate that by writing your own plugin for it.

If you don't find any massive explosive issues with it, mind if I close the PR and issue?

DRSDavidSoft commented:
I don't mind at all, but IMO it's more appropriate if you push it to the 1.0 branch for historical versions (some future visitors might still be wanting to use the older version of kint.)

It's just an example script, after all.

jnvsor commented:
Ah true, I thought this was tweaking behaviour not an example. Though I do spy a bug right off: The day in the dates will go up to `2013-01-011` now. Lower the condition to 10 or so and it should be fine

DRSDavidSoft committed:
Fixed the date padding

Used PHP's internal `str_pad` function.

DRSDavidSoft commented:
You have eagle eyes! I hope you accept this solution, I tried to keep it still readable.

jnvsor commented on examples/overview.php:
```diff
@@ -115,16 +115,17 @@ public function ensureUser() { $this->user->ensure(); }
 
-for ( $i = 1; $i < 6; $i++ ) {
+for ( $i = 1; $i < 12; $i++ ) {
+	$number = str_pad($input, 2, "0", STR_PAD_LEFT);
```
I spy with my little eye, copy pastey pastey :)

jnvsor replied:
In case you still don't see it: `$i != $input`

DRSDavidSoft replied:
That's actually a misplaced auto completion. That's what happens if I hurry to commit :D

DRSDavidSoft committed:
Fixed a variable bug

This bug fix removes a serious problem in which the `$i` is mis-interpreted as the `$input` variable.

DRSDavidSoft reviewed (commented):
Done. Sorry about multiple commits!

DRSDavidSoft commented:
Everything good for the PR now?

jnvsor commented:
No problem - I'll squash them in the end. Just let me test this quickly

jnvsor commented:
Squashed as 425c6127b54e47934f36d69519342c94d2d1c911
